Lannister/Data/Mappers/TransactionDto.swift

```diff
@@ -18,7 +18,7 @@ class TransactionDto : NSObject {
         transaction.name = managedObject.name
         transaction.type = managedObject.type
         transaction.value = managedObject.value
-        transaction.holding = HoldingDto().holding(from: managedObject.holding!)
+//        transaction.holding = HoldingDto().holding(from: managedObject.holding!)
```

Comment:
Remove this as well since it's no longer needed?

Reply:
👍